junit-jupiter-api/src/main/java/org/junit/jupiter/api/Order.java

```diff
@@ -23,8 +23,8 @@
 
 /**
  * {@code @Order} is an annotation that is used to configure the
- * {@linkplain #value order} in which the annotated element (i.e., field or
- * method) should be evaluated or executed relative to other elements of the
+ * {@linkplain #value order} in which the annotated element (i.e., field,
+ * method or class) should be evaluated or executed relative to other elements of the
```

Comment:
Please hard-wrap at 80 chars.

Reply:
I trusted spotless blindly on this, sorry